<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="track_search_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="track_search_bar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter track (e.g. Thwomp Ruins or TR): </string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>531</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="data_layout"/>
    </widget>
   </widget>
   <widget class="QLineEdit" name="score_input">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter placement (1-12):</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="track_info">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="overall_info">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ownership_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Created by Bao Phung.  Credit for track icons goes to the Super Mario Wiki fan page.  Mainly used to self-evaluate performance on each of the 96 available tracks in MK8D.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="frequency_button">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>400</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Frequency Graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
